assets/tests/components/routeLadder.test.tsx

```diff
@@ -259,6 +259,89 @@ describe("routeLadder", () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test("renders descriptive content in dropdown if there's a non-skate alert", async () => {
+    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetoursPilot])
+
+    const route: Route = routeFactory.build({
+      id: "28",
+      name: "28",
+    })
+    const timepoints = [
+      { id: "MATPN", name: "MATPN Name" },
+      { id: "WELLH", name: "WELLH Name" },
+      { id: "MORTN", name: "MORTN Name" },
+    ]
+
+    render(
+      <RouteLadder
+        route={route}
+        timepoints={timepoints}
+        vehiclesAndGhosts={undefined}
+        selectedVehicleId={undefined}
+        deselectRoute={() => {}}
+        reverseLadder={() => {}}
+        toggleCrowding={() => {}}
+        ladderDirections={{}}
+        ladderCrowdingToggles={{}}
+        hasAlert={true}
+      />
+    )
+
+    act(() => {
+      fireEvent.click(screen.getByRole("button", { name: "28 Route Options" }))
+    })
```

Comment:
```suggestion
    await userEvent.click(
      screen.getByRole("button", { name: "28 Route Options" })
    )
```

When I was playing around with this on my computer, I ran into some of those annoying `An update occurred outside of act` errors. This seemed to go away when using `userEvent` instead of `fireEvent` (I don't really know why, but I do remember other reasons that `userEvent` is supposed to be better than `fireEvent`, so I try to use it when possible)

Reply:
Great! Thank you